sdk/python/kfp/dsl/_placeholders.py

```diff
@@ -0,0 +1,13 @@
+FLAGS = {
+			'EXECUTION_ID' : '{{workflow.uid}}-{{pod.name}}',
+			'POD_NAME': '{{pod.name}}',
+			'RUN_ID' : '{{workflow.uid}}',
```

Comment:
It'd be better if you used `{{workflow.labels.pipeline/runid}}` instead.

Please, see kubeflow/pipelines#3709 (and the linked issues) for more context.

Reply:
thanks for the feedback! I'm happy to use whatever makes most sense. Reading through the issue, have you guys been able to confirm that it is the answer to this question?
`So, what we need is a placeholder that stays stable when retrying, but changes when cloning`

Reply:
Let's keep the two PRs independent for now.